### PR TITLE
Fix instructions in readme file to build R interface

### PR DESCRIPTION
**Pull request by TolisChal**
This PR solves the issue https://github.com/GeomScale/volume_approximation/issues/62.

### CODE REVIEW

File: doc/r_interface.md [hunk 1]
```diff
@@ -3,16 +3,18 @@
 
 ####  Install Rcpp package  
  
-* Install package-dependencies: `Rcpp`, `RcppEigen`, `BH`, `lpSolveAPI`.  
+* Install package-dependencies: `Rcpp`, `RcppEigen`, `BH`.  
 
-1. Then use devtools package to install `volesti` Rcpp package. In folder /root/R-proj Run:
+1. Then use devtools package to install `volesti` Rcpp package. From terminal go to folder /root/R-proj and run in `R`:
```
Comment (vissarion) [inline doc/r_interface.md]: why not just `Rscript -e 'Rcpp::compileAttributes()'`

Comment (vissarion) [reply doc/r_interface.md]: without entering and exiting `R`

File: doc/r_interface.md [hunk 2]
```diff
@@ -3,16 +3,18 @@
...
 Rcpp::compileAttributes()  
-library(devtools)  
-devtools::build()  
-devtools::install()  
-library(volesti)  
+```  
+  
+Then, exit `R` and run:  
 ```
+R CMD INSTALL --no-multiarch --with-keep.source R-proj
```
Comment (vissarion) [inline doc/r_interface.md]: we are in `root/R-proj` so the correct command is `R CMD INSTALL --no-multiarch --with-keep.source .`

**Review by vissarion — CHANGES_REQUESTED** [review]

File: doc/r_interface.md (hunk 2 above)
Comment (TolisChal) [reply doc/r_interface.md]: Thanks for this comment. Done.

File: doc/r_interface.md (hunk 1 above)
Comment (TolisChal) [reply doc/r_interface.md]: Thank you @vissarion! Done.

**Review by vissarion — APPROVED** [review]

**Pull request merged**